modules/saas/apps/saml-saas/src/main/java/com/liferay/saml/saas/internal/configuration/SaasConfiguration.java

```diff
@@ -24,18 +24,19 @@
 @ExtendedObjectClassDefinition(
 	generateUI = false, scope = ExtendedObjectClassDefinition.Scope.COMPANY
 )
-@Meta.OCD(id = "com.liferay.saas.configuration.SaasConfiguration")
+@Meta.OCD(id = "com.liferay.saml.saas.configuration.SAMLSaasConfiguration")
 public interface SaasConfiguration {
 
 	@Meta.AD(
-		deflt = "false", id = "saas.production.environment", required = false
+		deflt = "false", id = "saml.saas.production.environment",
+		required = false
 	)
 	public boolean productionEnvironment();
 
-	@Meta.AD(id = "saas.pre.shared.key", required = false)
+	@Meta.AD(id = "saml.saml.saas.pre.shared.key", required = false)
```

Comment:
@csierra Did you want "saml" twice here?

Reply:
nop... typo